src/app/shared/pipes/duration-pipe.pipe.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { DurationPipePipe } from './duration-pipe.pipe';
+
+describe('DurationPipePipe', () => {
+  it('create an instance', () => {
+    const pipe = new DurationPipePipe();
+    expect(pipe).toBeTruthy();
+  });
```

Comment:
Also you can test the pipe something like this 
`const pipe = new DurationPipePipe();
expect(pipe.transform(30)).toBe('30min');`

Reply:
super easy after your comment)